### DISCUSSION TITLE
Porting to Sub1GHz boards

### DISCUSSION THREAD

**Discussion by leoheck** [Koenkk/zigbee2mqtt#7082] [Devices]
Hey, @koenkk do you know how hard is to port this zigbee2mqqtt project to work using these sub1ghz boards from TI? My idea is to have the same thing but running on this Simplelink CC1352.

**Comment by Koenkk** [comment]
Zigbee2MQTT isn't aware on the frequency on which the network is running. I believe it's just a matter of flashing a sub 1ghz firmware on the sniffer.
